test/parallel/test-https-agent-getname.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+require('../common');
+var assert = require('assert');
+var https = require('https');
+
+var agent = new https.Agent();
+
+// empty options
+assert.strictEqual(
+  agent.getName({}),
+  'localhost:::::::::'
+);
+
+// pass all options arguments
+var options = {
```

Comment:
Done. 
Also modified the port from `80` to `443`, though it makes no difference to the test.